USER_MANUAL.md

```diff
@@ -300,7 +307,93 @@ Note that programs are interpreted as a flat list of parameters that are incorpo
 
-_TODO_
+### &lt;input_directory&gt;
+
+**Data type:** String
+
+**Default input:** "./inputs"
+
+**Description:** Specifies the folder/directory containing virtual world defining files like the[emissions_file](#emissions_file), [sites_file](#sites_file), etc. Accepts either an absolute path or a relative path from the root folder.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** N/A
+
+### &lt;output_directory&gt;
+
+**Data type:** String
+
+**Default input:** './outputs'
+
+**Description:** Specifies the folder/directory to generate output files into. Accepts either an absolute path or a relative path from the root folder.
+
+**Notes on acquisition:** It is recommended for the `output_directoy` be specified for each simulation that is ran.
+
+**Notes of caution:** The contents of the existing folder is **removed** and **overwritten**. Rename folders to ensure that old output files are not lost.
+
+### &lt;baseline_program&gt;
+
+**Data type:** String
+
+**Default input:** 'P_none'
+
+**Description:** Refers to a [name of a program](#program_name) that is a part of the simulation. Results requiring a reference point for comparison, such as mitigation efforts, will be derived by comparing the output values from this program.
+
+**Notes on acquisition:** Typically a program that represents a scenario where there is no formal LDAR or that has no LDAR method is recommended as the baseline program. Simply put, create a program with no methods.
+
+**Notes of caution:** A baseline program is required to successfully run the simulation.
+
+### &lt;reference_program&gt; WIP
+
+**Data type:** String
+
+**Default input:** 'P_OGI'
+
+**Description:** Refers to a [name of a program](#program_name), which alternative programs are compared.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** N/A
+
+### &lt;processes_count&gt;
+
+**Data type:** Integer (Numeric)
+
+**Default input:** 6
+
+**Description:** The maximum number of parallel tasks or processes that the simulator can use simultaneously. To simplify, this is the limit of how many different tasks the simulator can handle at a given time.
+
+**Notes on acquisition:** In general, many modern computers can effectively handle around 6 concurrent processes without significant performance issues. This number is influenced by factors such as the computer's hardware specifications, operating system efficiency, and the resource demands of the individual processes.
+
+**Notes of caution:** Python is limited and cannot achieve true simultaneous execution of Python code. Therefore, generating too many processes may cause slower performance due to excessive context switching between the different processes.
+
+A minimum of a single process is required for the simulation to run.
+
+### &lt;simulation_count&gt;
+
+**Data type:** Integer (Numeric)
+
+**Default input:** 2
+
+**Description:** The quantity of simulation iterations to execute for the given programs.
```

Comment:
Should we say simulation "rounds" here to be consistent with instructional video nomenclature?